examples/theming/src/app/global.css

```diff
@@ -0,0 +1,7 @@
+@import '@primer/primitives/dist/css/functional/themes/light.css';
+@import '@primer/primitives/dist/css/functional/themes/dark.css';
```

Comment:
We might want to include other primitives like spacing font etc. as far as color themes I think light/dark are the bare minimum and probably fine for this example

Reply:
You could pull in this index file for everything other than color https://unpkg.com/browse/@primer/primitives@8.2.2/dist/css/primitives.css